### DISCUSSION TITLE
Different model sizes of Basicvsr

### DISCUSSION THREAD

**Discussion by zhangxiaobaibai** [open-mmlab/mmediting#1210] [Models] [answered]
Hi, 

I trained Basicvsr on Vimeo90k and the size of saved model is 72.35 MB, which is quite different from your provided model in model zoo(24.06 MB).

**Comment by ckkelvinchan** [comment, marked as answer]
The saved model contains the optimizer so that you can resume training. But the model we released does not contain such info. So it is smaller.
